src/octoprint/plugins/virtual_printer/__init__.py

```diff
@@ -136,12 +136,18 @@ def register_transport_hook(self, *args, **kwargs):
         ]
 
     def cli_commands(self, cli_group, pass_octoprint_ctx, *args, **kwargs):
+        import fcntl
         import os
+        import pty
         import select
         import socket
+        import stat
+        import sys
+        import termios
         import threading
```

Comment:
`fcntl`, `pty`, `termios` need to move into a posix only section or this will make the whole thing fail hard on windows systems

Reply:
Fixed